From 5cd60718c23839f9ba639b4063545cb0c97118cf Mon Sep 17 00:00:00 2001
From: Lin Xie <lin.xie@intel.com>
Date: Wed, 27 Mar 2013 16:29:14 +0800
Subject: Add a new flag HWC_ROTATION_IN_PROGRESS.

The flag is set by SurfaceFlinger to indicate that the
window system is still in the progress of a rotation.

Issue: 82548, ADEV-107
Category: Device Enablement
Domain: AOSP-libhardware
Upstream-candidate: Yes
Source: Internal
Signed-off-by: Lin Xie <lin.xie@intel.com>
Change-Id: I7ef990b8e3e8ee56fb9f3ec1c8e770495f4d3729
Reviewed-on: http://android.intel.com:8080/98577
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Zhu, Tianyang <tianyang.zhu@intel.com>
Reviewed-by: Wang, Lili A <lili.a.wang@intel.com>
Reviewed-by: Veeramani, Karthik <karthik.veeramani@intel.com>
Reviewed-by: Shi, PingX <pingx.shi@intel.com>
Tested-by: Shi, PingX <pingx.shi@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/hardware/hwcomposer.h |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/include/hardware/hwcomposer.h b/include/hardware/hwcomposer.h
index 86479d3..9f516dc 100644
--- a/include/hardware/hwcomposer.h
+++ b/include/hardware/hwcomposer.h
@@ -284,6 +284,12 @@ enum {
      * and acquire fences.
      */
     HWC_GEOMETRY_CHANGED = 0x00000001,
+
+    /*
+     * HWC_ROTATION_IN_PROGRESS is set by SurfaceFlinger to indicate that the
+     * window system is still in the progress of a rotation.
+     */
+    HWC_ROTATION_IN_PROGRESS = 0x00000010,
 };
 
 /*
-- 
1.7.4.1

